endoreg_db/management/commands/import_video.py

```
@@ -6,6 +6,7 @@
 This command is designed to be run from the command line and takes various arguments
 to specify the video file, center name, and other options.
 """
+from multiprocessing import process
```

Ruff (0.11.9)

`multiprocessing.process` imported but unused

Remove unused import: `multiprocessing.process`

(F401)